codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSServiceUtils.kt

```diff
@@ -7,24 +7,25 @@ package software.amazon.smithy.aws.swift.codegen
 
 import software.amazon.smithy.aws.traits.ServiceTrait
 import software.amazon.smithy.model.shapes.ServiceShape
-import software.amazon.smithy.swift.codegen.model.expectTrait
+import software.amazon.smithy.swift.codegen.model.getTrait
 
 /**
  * Get the [sdkId](https://smithy.io/2.0/aws/aws-core.html#sdkid) from the (AWS) service shape
+ * or return a default value if the trait is not present.
  */
 val ServiceShape.sdkId: String
-    get() = expectTrait<ServiceTrait>().sdkId
+    get() = getTrait<ServiceTrait>()?.sdkId ?: "defaultSdkId"
```

Comment:
default values are necessary since not all protocol tests will be tied to a particular AWS service. This is the first set of smithy files we need to codegen where there is no service trait

Reply:
Is this because RPCv2 CBOR protocol tests are run in aws-sdk-swift instead of smithy-swift? Should these be included in the RPC protocol relocation ticket you create as changes that need to be reverted?

Reply:
nope nothing to do with that -- the protocol tests themselves (smithy files) do not tie these tests to a particular AWS service (no service trait). Smithy team did that on purpose because it is not an "aws" protocol

Reply:
Hmm I see; even if the RPC v2 CBOR generator was located in smithy-swift, we would be running RPCv2 protocol tests along with other protocol tests in aws-sdk-swift side.